package.json

```diff
@@ -49,6 +49,7 @@
     "@patternfly/react-tokens": "5.1.1",
     "date-fns": "2.30.0",
     "docker-names": "1.2.1",
+    "ipaddr.js": "^2.1.0",
```

Comment:
Strict versions please. I'm not sure if this breaks dependabot, but npm installs should be reproducible.